.circleci/config.yml

```diff
@@ -38,8 +38,31 @@ workflows:
           requires:
             - build
 
+      - hold-push-cluster-operator-to-aliyun-pr:
+          type: approval
+          requires:
+            - build
+          # Needed to prevent job from being triggered on master branch.
+          filters:
+            branches:
+              ignore: master
+
+      - architect/push-to-docker:
+          name: push-cluster-operator-to-aliyun-pr
+          image: "registry-intl.cn-shanghai.aliyuncs.com/giantswarm/cluster-operator"
+          username_envar: "ALIYUN_USERNAME"
+          password_envar: "ALIYUN_PASSWORD"
+          # Push to Aliyun should execute for non-master branches only once manually approved.
+          requires:
+            - hold-push-cluster-operator-to-aliyun-pr
+          # Needed to prevent job from being triggered on master branch.
+          filters:
+            branches:
+              ignore: master
```

Comment:
I don't think we need the filter because approval won't work on other branches anyway. But this is a minor thing.